src/components/modal/addSrtModal/AddSrtModal.tsx

```diff
@@ -0,0 +1,290 @@
+              {ingests.map((ingest) => (
+                <option key={ingest.uuid} value={ingest.name}>
+                  {ingest.name}
+                </option>
+              ))}
+            </select>
+            {isIngestNameError && (
+              <p className="text-xs text-button-delete mt-2">
+                {t('inventory_list.no_ingest_selected')}
+              </p>
+            )}
+          </span>
+        </span>
+        <span className="flex items-center w-full">
+          <h2 className="flex-none w-1/3 text-left">
+            {t('inventory_list.name')}:
+          </h2>
+          <span className="flex flex-col w-full items-center">
+            <input
+              className={`w-full ${
+                isNameError ? 'border-error' : 'border-gray-600'
+              } flex-grow w-full ml-2 border text-sm rounded-lg pl-2 pt-1 pb-1 bg-gray-700 placeholder-gray-400 text-white focus:ring-blue-500 focus:border-blue-500`}
+              type="text"
+              value={name}
+              onChange={handleInputChange(setName, setIsNameError)}
+            />
+            {isNameError && (
+              <p className="text-xs text-button-delete mt-2">
+                {t('inventory_list.no_name')}
+              </p>
+            )}
+          </span>
+        </span>
+        <span className="flex items-center w-full">
+          <h2 className="flex-none w-1/3 text-left">
+            {t('inventory_list.local_ip')}:
+          </h2>
+          <input
+            className="flex-grow ml-2 border text-sm rounded-lg pl-2 pt-1 pb-1 bg-gray-700 border-gray-600 placeholder-gray-400 text-white focus:ring-blue-500 focus:border-blue-500"
+            type="text"
+            value={localIp}
+            onChange={handleInputChange(setLocalIp)}
+          />
+        </span>
+        <span className="flex items-center w-full">
+          <h2 className="flex-none w-1/3 text-left">
+            {t('inventory_list.local_port')}:
+          </h2>
+          <input
+            className="flex-grow ml-2 border text-sm rounded-lg pl-2 pt-1 pb-1 bg-gray-700 border-gray-600 placeholder-gray-400 text-white focus:ring-blue-500 focus:border-blue-500"
+            type="number"
+            value={localPort}
+            onChange={handleInputChange(setLocalPort)}
+          />
+        </span>
+        <div
+          className={`${styles.expandableSection} ${
+            mode === 'Caller' ? styles.expanded : styles.collapsed
+          } space-y-4 w-full`}
+        >
+          <span className="flex items-center w-full">
+            <h2 className="flex w-1/3 text-left">
+              {t('inventory_list.remote_ip')}:
+            </h2>
+            <input
+              className="flex-grow ml-2 border text-sm rounded-lg pl-2 pt-1 pb-1 bg-gray-700 border-gray-600 placeholder-gray-400 text-white focus:ring-blue-500 focus:border-blue-500"
+              type="text"
+              value={remoteIp}
```

Comment:
A lot of repetition with the CSS. Maybe we should look into using Custom tailwind classes with @apply